### DISCUSSION TITLE
Radiomaster Boxer EdgeTx 2.8.2

### DISCUSSION THREAD

**Discussion by MacFerran** [EdgeTX/edgetx#3496] [General]
I have updated to the EdgeTx 2.8.2 version and the color of the central light has changed before it was blue and now it is green, is it for some reason?

**Comment by pfeerick** [comment]
Probably because you were originally running a pre-release version of 2.8.0, just before the colors were corrected? It is supposed to be blue while starting up, and then green once running (and no errors, else it's red), on radios that have RGB power leds (unless a manufacturer requests otherwise).
